### ISSUE TITLE
Procedurals does not inherit tranformations from its hirarchy

### ISSUE THREAD

**Issue by kreppene** [Autodesk/arnold-usd#546]
**Describe the bug**
When a ArnoldProcedurals is parented under a hierarchy , and the hierarchy transformations move,
the ArnoldProcedural does not follow the transformations.

**Example**
```
/Root
        /offest
           |- xformOp:translate (-144.3573998251817, 17.323716548862503, -4.389528375776521)
           |- xformOpOrder [xformOp:translate, xformOp:rotateXYZ, xformOp:scale]
                 /ArnoldProcedural (This node does not move)
```
               

**To Reproduce**
standin.usda
Export a standin.usda file with this nodeName /Root/offset/myStandin, the stanin should be pointing to a simple test geo.

  anim.usda
Then from maya or simular create the same hirarchy /Root/offset/myStandin , move and rotate the  /Root node around, and export the hirarchy to anim.usd

renderme.usda
Now just layer the 2 usda files,  anim.usda and then standin.usda at the top
```
#usda 1.0
(
    subLayers = [
        @./standin.usda@,
        @./anim.usda@
    ]
)


```
when rendering the renderme.usda file you can se the standin does not follow our animation

**Expected behavior**
Expect the standind to move the same way as the cube in the gif attached

**Screenshots**
This shows the viewport, but the same behaviour happens inn render
![2020-09-24_14-11-46](https://user-images.githubusercontent.com/7374105/94143373-07ddf380-fe70-11ea-949d-84b7a0d159de.gif)

**Used Software Versions**
 - Arnold: 6.0.4.0
 - USD: 0.20.05
 - Compiler: VS
 - OS: Windows 10
- Maya / mtoa

i attached a repro , but you may need to change the path to the standing for it work
[arnold_procedural.zip](https://github.com/Autodesk/arnold-usd/files/5275906/arnold_procedural.zip)

**Comment by sebastienblor** [comment]
If I understand well, what happens here is that `ArnoldProcedural` is a custom schema that doesn't inherit from `UsdGeomXformable`, so it doesn't know anything about usd xforms from its hierarchy

**Comment by sebastienblor** [comment]
This was fixed in #627, we had apparently a duplicate issue #583
I will mark the issue as closed, we can reopen or create new tickets if needed
